netket/models/autoreg.py

```diff
@@ -44,8 +45,8 @@ def conditionals(self, inputs: Array, cache: PyTree) -> Tuple[Array, PyTree]:
 class ARNNDense(ARNN):
     """Autoregressive neural network with dense layers."""
```

Comment:
Can you say something more in this docstring? 
As far as I knew an ARNN is made up of several masked convolutions. Why are we using dense here?

Also, can you write a bunch of formulas about how this acts and what are the most important things to tune? 
See examples in `GCNN`.

Reply:
In general, ARNN only requires that 'any spin only depends on previous spins', so a dense layer with a triangular mask is a very general form, and masked conv layers are special cases

I'll write more in the docstring.